test/sanity/pep8/legacy-files.txt

```diff
@@ -375,4 +375,3 @@ lib/ansible/modules/system/systemd.py
 lib/ansible/modules/system/timezone.py
 lib/ansible/modules/system/ufw.py
 lib/ansible/modules/system/user.py
```

Comment:
You forgot to remove `user.py` from legacy-files.txt

Reply:
no, it is not completely done, therefore i left it as it is.